published/012-2016-August-24.md

```diff
@@ -0,0 +1,110 @@
+- [Milestone 5 - 1st implementation Peer Routing + Content Routing (the go-ipfs DHT)](https://github.com/ipfs/js-ipfs/blob/master/ROADMAP.md#milestone-5---1st-implementation-peer-routing--content-routing-the-go-ipfs-dht)
+- [Milestone 6 - The last miles for complete feature parity with go-ipfs](https://github.com/ipfs/js-ipfs/blob/master/ROADMAP.md#milestone-6---the-last-miles-for-complete-feature-parity-with-go-ipfs)
+
+The [js-ipfs-api](https://github.com/ipfs/js-ipfs-api) v7 has now been shipped; a full exposition of this and the thinking behind it can be found [here](https://github.com/ipfs/js-ipfs-api/pull/305). The short story is that it now has 30% more files interfacing, which means more interop with go-ipfs and more you can do with it. For future changes, [follow this issue](https://github.com/ipfs/interface-ipfs-core/issues/55), which will communicate interface changes on js-ipfs and js-ipfs-api.
+
+Moving to pull-streams: The 🚲🏠 (bike-shedding) was strong in this one, but we've finally made a call on the Control Flow library to use in order to solve the hiccups we have had in the past with Node.js Streams. This discussion took a lot of input from several people familiar with the js-ipfs project and the contenders for the Control Flow Library. You can read the whole thread [here](https://github.com/ipfs/js-ipfs/issues/362).
+
+We are working on better changelogs for our releases, which will be automatically generated. We've finalized our commit message convention, [here](https://github.com/ipfs/community/blob/master/js-commit-guidelines.md), so that they are easily parsable and human understandable. The issue to track the development of the tooling required to add this feature to our release process is [here](https://github.com/dignifiedquire/aegir/issues/30#issuecomment-236685903).
+
+The js-ipfs CLI also got an upgrade. We moved from ronin to yargs, a more supported and used CLI toolkit that enables us to have menus for the js-ipfs CLI just like go-ipfs has.
+
+On a small note: js-ipfs, like all of the IPFS projects, is heavily modular, so subscribing to a single repository is not always the optimal way to get updates. We tend to refactor out anything small enough to be its own meaningful module.
+
+For a full discussion about where we are with js-ipfs, please check out @diasdavid's [Captain's Log](https://github.com/ipfs/js-ipfs/issues/30#issuecomment-239642234)
+
+### [libp2p](https://github.com/libp2p)
+
+We have recently created the [libp2p](https://github.com/libp2p) organization, with the goal of building a modularized and extensible network stack to overcome the networking challenges faced when doing peer-to-peer applications. libp2p is used by IPFS as its networking library. Libp2p-land comprises both JavaScript and Go libraries, and already has dozens of repos. You can check out the main repository, describing our efforts, here: [libp2p/libp2p](https://github.com/libp2p).
```

Comment:
Libp2p-land makes my imagined reader confused :P i think "Libp2p comprises both [...]" is fine

also the link doesn't point to the libp2p/libp2p repo